src/foundry/common/utils/helpers.d.mts

```diff
@@ -304,7 +304,7 @@ export function isNewerVersion(v1: number | string, v0: number | string): boolea
  * @param value - The value to test
  * @returns Is the value empty-like?
  */
-export function isEmpty(value: undefined | null | unknown[] | object | Set<unknown> | Map<unknown, unknown>): boolean;
+export function isEmpty(value: unknown): boolean;
```

Comment:
Can you switch to:
```ts
export function isEmpty(value: undefined | null | unknown[] | object | Set<unknown> | Map<unknown, unknown> | NonNullish): boolean;
```

I think there's more value in the verbose signature. Because of `undefined | null | NonNullish` it is going got accept anything, just like `unknown`, but this explicit iteration helps show what values actually have a sane `isEmpty` implementation.